src/main/kotlin/org/openbroker/no/privateunsecuredloan/events/Message.kt

```diff
@@ -0,0 +1,13 @@
+package org.openbroker.no.privateunsecuredloan.events
+
+import org.openbroker.common.model.Reference
+
+/**
+ * An event containing information on the loan process
+ */
+
+data class Message @JvmOverloads constructor(
```

Comment:
I do not think we need `@JvmOverloads` annotation if the constructor has no default values on arguments.

Reply:
Thanks, this has been changed now.